macros/calibrateCTSEvents.C

```diff
@@ -0,0 +1,313 @@
+        case 28: totCalib=ToT*m_layer3[2]+y0_layer3[2]; break;
+        case 29: totCalib=ToT*m_layer3[30]+y0_layer3[30]; break;
+        case 30: totCalib=ToT*m_layer3[1]+y0_layer3[1]; break;
+        case 31: totCalib=ToT*m_layer3[31]+y0_layer3[31]; break;
+        case 32: totCalib=ToT*m_layer3[0]+y0_layer3[0]; break;   
+        }
+      }else if (int(layer) == 4){
+      switch(int(y)) {
+        case 1: totCalib=ToT*m_layer4[0]+y0_layer4[0]; break;
+        case 2: totCalib=ToT*m_layer4[31]+y0_layer4[31]; break;
+        case 3: totCalib=ToT*m_layer4[1]+y0_layer4[1]; break;
+        case 4: totCalib=ToT*m_layer4[30]+y0_layer4[30]; break;
+        case 5: totCalib=ToT*m_layer4[2]+y0_layer4[2]; break;
+        case 6: totCalib=ToT*m_layer4[29]+y0_layer4[29]; break;
+        case 7: totCalib=ToT*m_layer4[3]+y0_layer4[3]; break;
+        case 8: totCalib=ToT*m_layer4[28]+y0_layer4[28]; break;
+        case 9: totCalib=ToT*m_layer4[4]+y0_layer4[4]; break;
+        case 10: totCalib=ToT*m_layer4[27]+y0_layer4[27]; break;
+        case 11: totCalib=ToT*m_layer4[5]+y0_layer4[5]; break;
+        case 12: totCalib=ToT*m_layer4[26]+y0_layer4[26]; break;
+        case 13: totCalib=ToT*m_layer4[6]+y0_layer4[6]; break;
+        case 14: totCalib=ToT*m_layer4[25]+y0_layer4[25]; break;
+        case 15: totCalib=ToT*m_layer4[7]+y0_layer4[7]; break;
+        case 16: totCalib=ToT*m_layer4[24]+y0_layer4[24]; break;
+        case 17: totCalib=ToT*m_layer4[8]+y0_layer4[8]; break;
+        case 18: totCalib=ToT*m_layer4[23]+y0_layer4[23]; break;
+        case 19: totCalib=ToT*m_layer4[9]+y0_layer4[9]; break;
+        case 20: totCalib=ToT*m_layer4[22]+y0_layer4[22]; break;
+        case 21: totCalib=ToT*m_layer4[10]+y0_layer4[10]; break;
+        case 22: totCalib=ToT*m_layer4[21]+y0_layer4[21]; break;
+        case 23: totCalib=ToT*m_layer4[11]+y0_layer4[11]; break;
+        case 24: totCalib=ToT*m_layer4[20]+y0_layer4[20]; break;
+        case 25: totCalib=ToT*m_layer4[12]+y0_layer4[12]; break;
+        case 26: totCalib=ToT*m_layer4[19]+y0_layer4[19]; break;
+        case 27: totCalib=ToT*m_layer4[13]+y0_layer4[13]; break;
+        case 28: totCalib=ToT*m_layer4[18]+y0_layer4[18]; break;
+        case 29: totCalib=ToT*m_layer4[14]+y0_layer4[14]; break;
+        case 30: totCalib=ToT*m_layer4[17]+y0_layer4[17]; break;
+        case 31: totCalib=ToT*m_layer4[15]+y0_layer4[15]; break;
+        case 32: totCalib=ToT*m_layer4[16]+y0_layer4[16]; break;
+      }      
+    }
+    signal = Signal(totCalib,timeCalib,signalNr,chID,layer,TDC,padiwaConfig);
+    module.addSignal(signal);
+
+    event->setEventNr(ULong_t(eventNr));
+    event->setPadiwaConfig(UShort_t(padiwaConfig));
+
+    prevEventNr = ULong_t(eventNr);
+  }
+
+  tree->Write("data");
+  fout->Close();
+
+  delete event;
+  event=nullptr;
+}
+
+int main(int argc, char** argv)
+{
+  char    inputFile[512]="";
+  char    outputFile[512]="calibratedCTSEvents.root";
```

Comment:
I would prefer to have all default output names be `macroName_output.root`. Then one needs 0 thought when the `-o` option is not used while debugging.